pkg/kudoctl/cmd/init_test.go

```diff
@@ -92,43 +94,46 @@ func TestInitCmd_output(t *testing.T) {
 	MockCRD(c.client, "certificates.cert-manager.io", "v1alpha2")
 	MockCRD(c.client, "issuers.cert-manager.io", "v1alpha2")
 
-	tests := []string{"yaml"}
+	tests := []output.Type{output.TypeYAML, output.TypeJSON}
 	for _, s := range tests {
```

Comment:
super nit:
```suggestion
	for _, tt := range tests {
```

Reply:
Well, as i'm already modifying this file I might as well change this :)